### ISSUE TITLE
BUG - Accessibility internal Tests FAIL

### ISSUE THREAD

**Issue by peteCodes4u** [Quartzion/qts-v1.3.0#43]
**Actual** When executing the internal accessibility tests the tests fail to run and return the following error.
```ruby
 FAIL  ./accessibility.test.jsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation, specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    C:\Users\peter\QTS\qts-v1.3.0\client\node_modules\react-markdown\index.js:10
    export {
    ^^^^^^

    SyntaxError: Unexpected token 'export'

      1 | import React from 'react';
    > 2 | import ReactMarkdown from 'react-markdown'
        | ^
      3 | import slugify from "slugify";
      4 | import { Button } from "react-bootstrap";
      5 | import ConnectWithUs from '../components/ConnectWithUs';

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1314:40)
      at Object.require (client/src/utils/cardUtils.jsx:2:1)
      at Object.require (client/src/components/Blogs/index.jsx:7:1)
      at Object.require (accessibility.test.jsx:31:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        3.525 s
Ran all test suites.
```
**Expected** All Tests Run as expected without errors and return passing results for all components and pages.

<img width="997" height="750" alt="Image" src="https://github.com/user-attachments/assets/eae3720c-44f9-4ec3-96d3-f0abd6e95ab3" />

**Comment by peteCodes4u** [comment]
This ticket is being closed. A new ticket to address the services component is going to handle the uselocation issue

#45 

the changes for this ticket can be found in the following branch:
[resolve-bug-43](https://github.com/Quartzion/qts-v1.3.0/tree/resolve-bug-43)
